app/views/assessments/_install_assessment_table.html.erb

```diff
@@ -45,8 +45,8 @@
               <b class="install-failure-p">
                 Failed to Install
               </b>
-              <label>
-                <%= check_box_tag asmt_file, asmt_file, true, class: 'cbox', id: "#{asmt_file}_checkbox" %>
+              <label class="assessment-checkbox">
```

Comment:
For some reason this like appears kind of misaligned

![Screenshot 2024-10-06 at 1 02 13 PM](https://github.com/user-attachments/assets/a6323e6c-a8e5-4f4d-b0c0-4551ee11a15f)

Could we add some styling to make it more aligned: I tried this and I think something like this works for me
`.assessment-checkbox {
  display: flex;
  justify-content: center;
}`